test/integration/integration-tests.js

```diff
@@ -196,6 +197,16 @@ test('Gets v2 space for read only tests', (t) => {
     })
 })
 
+test('Create organization for tests which create, change and delete data', (t) => {
```

Comment:
This text should probably be changed to smth like 'Get organization for tests...' ?

Reply:
Sure thing, thx a lot. fixed as well